### DISCUSSION TITLE
Is torrent flags are not part of resume data saved?

### DISCUSSION THREAD

**Discussion by hellozyemlya** [arvidn/libtorrent#7675] [Q&A]
So, pause flag is not saved when I call torrent_handle.save_resume_data(lt.torrent_handle.only_if_modified | lt.torrent_handle.save_info_dict). Is that expected and flags preserving must be implemented on application layer?

**Comment by arvidn** [comment]
No, that's not expected. unless you're referring to the *session* being paused.
